packages/config/config/devices/0x027a/zen77.json

```diff
@@ -155,6 +155,26 @@
 			"#": "26",
 			"$if": "firmwareVersion >= 10.0",
 			"$import": "templates/zooz_template.json#local_programming"
+		},
+		{
+			"#": "27",
+			"$if": "firmwareVersion >= 10.20",
```

Comment:
This (and further conditionals) should follow the following logic https://github.com/zwave-js/node-zwave-js/pull/5156#issuecomment-1280222835

```
IF version 1.0 and firmware > 10.20 then Parameter 27 - 30
IF version 2.0 and firmware > 2.10 then Parameter 27 - 30
IF version 3.0 and firmware > 3.0 then Parameter 27 - 30
```

Reason for that is Zooz changes the firmware version based on the hardware version